server/aap/commands/fulfill_image_assignments.py

```diff
@@ -120,10 +124,47 @@ def _get_outstanding_photo_assignments(self):
         :return: The Id of any sheduled picure assignments
         """
         service = superdesk.get_resource_service('assignments')
-        assignments = service.find(where={"assigned_to.state": ASSIGNMENT_WORKFLOW_STATE.ASSIGNED,
-                                          "planning.g2_content_type": "picture",
-                                          "planning.scheduled": {"$gte": utcnow() - timedelta(days=2)},
-                                          "lock_user": None})
+        query = {
+            'query': {
+                'filtered': {
```

Comment:
Don't use filtered/filter as this won't be supported in future elastic versions. Use query.bool.must/must_not